uint32 vehicles
vehicle[] vehicle_states